config/envs/development.py

```diff
@@ -5,14 +5,20 @@
 @configclass
 class DevelopmentConfig(DefaultConfig):
     ASSESSMENT_SCORES_ENDPOINT = (
-        DefaultConfig.ASSESSMENT_STORE_API_HOST
-        + "/assessments/{assessment_id}/sub_criterias/{sub_criteria_id}/scores"
+        DefaultConfig.ASSESSMENT_STORE_API_HOST + "/score"
     )
 
-    USE_LOCAL_DATA = True
-    # USE_LOCAL_DATA = False
+    # USE_LOCAL_DATA = True
+    USE_LOCAL_DATA = False
 
-    # FUND_STORE_API_HOST =
-    # "https://funding-service-design-fund-store-dev.london.cloudapps.digital"
-    # ASSESSMENT_STORE_API_HOST =
-    # "https://funding-service-design-assessment-store-dev.london.cloudapps.digital"
+    # FUND_STORE_API_HOST = "https://funding-service-design-fund-store-dev.london.cloudapps.digital" # noqa
+    # ASSESSMENT_STORE_API_HOST = "https://funding-service-design-assessment-store-dev.london.cloudapps.digital" # noqa
+
+    # for local dev/test use with flask run and USE_LOCAL_DATA = False:
+    FUND_STORE_API_HOST = (  # change in fund store repo and adjust here accordingly # noqa
+        "http://127.0.0.1:5002"
+    )
+    ASSESSMENT_STORE_API_HOST = (  # change in assessment store repo and adjust here accordingly # noqa
+        "http://127.0.0.1:5001"
+    )
```

Comment:
Is this is specific to your own local environment I don't know if it's a good idea to commit it to the repo, might break some peoples local environments if they have those apps running on slightly different ports or only accessible with a `host.docker.internal` address etc.